docker/README.md

```diff
@@ -21,7 +21,7 @@ The scripts are:
 A base image for each OS is built using:
```

Comment:
Lets add above that:
`run-all.sh` and `run-one.sh` tests both on jdk8 and 11 across OSes